dev_tools/notebook_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
I think we need more information about this, for people who follow in this footsteps and try to modify / augment this in the future.

Something akin to the PR description would be useful.

Reply:
Oh, I'll add that in then!!! Agreed on instructions - I think we should add something in the Development.MD as well!